### PR TITLE
Correct the device identifier of the window app 

### PR DESCRIPTION
**Pull request by danielsilabs**
Correct the device identified of endpoint 0 of the window app from the window covering id 514 (0x202) to the root device id 22 (0x16)

#### Problem
What is being fixed? 
* Device identifier for endpoint 0 is incorrect

#### Change overview
* correct device identifier and regenerate.

### CODE REVIEW

**Review by jmartinez-silabs — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/window-app/common/window-app.zap [hunk 1]
```diff
@@ -32,8 +32,8 @@
   "endpointTypes": [
     {
       "name": "MA-windowcovering",
-      "deviceTypeName": "MA-windowcovering",
-      "deviceTypeCode": 514,
+      "deviceTypeName": "MA-rootdevice",
```
Comment (jmeg-sfy) [inline examples/window-app/common/window-app.zap:35]: i never understood why there is this prefix MA ?

File: examples/window-app/common/window-app.zap [hunk 2]
```diff
@@ -1142,22 +1292,48 @@
...
+              "defaultValue": "en-US",
               "reportable": 1,
               "minInterval": 1,
               "maxInterval": 65534,
               "reportableChange": 0
             },
             {
-              "name": "SupportedCalendarTypes",
+              "name": "SupportedLocales",
               "code": 2,
```
Comment (jmeg-sfy) [inline examples/window-app/common/window-app.zap:1337]: @bzbarsky-apple how can we declare a list of supported locales in Zap directly ?

**Review by jmeg-sfy — APPROVED** [review]
Thanks for taking care of this !

**Review by tcarmelveilleux — APPROVED** [review]

File: examples/window-app/common/window-app.zap (hunk 1 above)
Comment (danielsilabs) [reply examples/window-app/common/window-app.zap:35]: @jmeg-sfy I believe that is just the prefix that was chose for Matter devices in the metadata. Similar to HA -> "Home Automation", ZLL -> "Zigbee Light Link", SE -> "Smart Energy" and so on..

File: examples/window-app/common/window-app.zap (hunk 2 above)
Comment (danielsilabs) [reply examples/window-app/common/window-app.zap:1337]: Boris can correct me if I am wrong, but I believe you would do this by doing a configure on the "Localization Configuration" and then entering into the text field  in the attributes UI.

Comment (jmeg-sfy) [reply examples/window-app/common/window-app.zap:1337]: ok this is unfortunate then we need to do tackle this is the code...

**Pull request merged**